File: 01_Language/01_Functions/java/src/main/java/com/onevgo/functions/IsDouble.java
package com.onevgo.functions;

public class IsDouble {
    public static boolean isDouble(Object var) {
        return var != null && var.getClass() == Double.class;
    }

    public static void main(String[] args) {
        System.out.println(isDouble(27.25));
        System.out.println(isDouble("abc"));
        System.out.println(isDouble(23));
        System.out.println(isDouble(23.5));
        System.out.println(isDouble(1e7));
        System.out.println(isDouble(true));
    }
}

File: 01_Language/01_Functions/python/is_double.py
# coding: utf-8

def is_double(var):
    return isinstance(var, float)


if __name__ == '__main__':
    print(is_double(27.75))
    print(is_double('abc'))
    print(is_double(23))
    print(is_double(23.5))
    print(is_double(1e7))
    print(is_double(True))
